source/overview/faq.rst

```diff
@@ -78,9 +78,9 @@ Are push notifications free?
 
 Yes, push notifications are free if you compile your own `push-proxy service <https://github.com/mattermost/mattermost-push-proxy>`_. Push notifications are also free if you use the hosted Test Push Notification Service (TPNS) provided by Mattermost, Inc.
 
-TPNS, hosted at ``http://push.mattermost.com``, doesn't offer transport-level encryption or production-level service level agreements (SLAs), so if you choose this option we recommend you configure your system to not include message contents in push notifications and not rely on the service for vital functions.
+TPNS, hosted at ``http://push.mattermost.com``, offers transport-level encryption, but not production-level service level agreements (SLAs).
```

Comment:
Should that not be https://push-test.mattermost.com?

Reply:
Good catch @JtheBAB! Updated